Gemfile

```diff
@@ -1,13 +1,13 @@
 source "https://rubygems.org"
 
-gem "rails", "~> 5.1.4"
+gem "coffee-rails", "~> 4.2"
+gem "jbuilder", "~> 2.5"
 gem "pg"
 gem "puma", "~> 3.7"
+gem "rails", "~> 5.1.4"
 gem "sass-rails", "~> 5.0"
+gem "tzinfo-data", platforms: %i[mingw mswin x64_mingw jruby]
```

Comment:
Not sure the extra gems should have been added in this PR as it's about Rubocop,
otherwise, all looks good!